modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -219,7 +218,7 @@ export default {
       const self = this;
       const enhanced = [];
       (styles || []).forEach(style => {
-        style.options = {};
+        style.options = { attrs: {} };
```

Comment:
so that we can apply bulk html attributes to elements. right now we only care about classes though